.idea/Front_for_DRF_filehosting.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should `.idea` folder be added to the repo?

Reply:
Delete .idea from repo